MaterialLib/MPL/Properties/pPengRobinson.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * \author Norbert Grunwald
+ * \date   18.09.2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#pragma once
+
+#include "../mpProperty.h"
+
+namespace MaterialPropertyLib
+{
+class Medium;
+class Phase;
+class Component;
+/**
+ * \class AverageVolumeFraction
+ * \brief A function averaging a property by volume fraction
+ * \details This property is usually a medium property, it
+ * computes the average of individual phase properties
+ * weighted by volume fraction.
+ */
+class PengRobinson final : public Property
```

Comment:
IMHO you could improve the documentation by giving a literature reference.

Reply:
You are right. I will add citations as soonest.